apps/studio/src/components/SearchableHeader.tsx

```diff
@@ -24,6 +24,7 @@ export const SearchableHeader = ({ siteId }: SearchableHeaderProps) => {
       py={{ base: 0, md: "0.5rem" }}
       px={{ base: 0, md: "0.5rem" }}
       background="white"
+      alignItems="start"
```

Comment:
super duper minor UI alignment issue but currently it's not well aligned for other searchableheader